spring-cloud-stream-schema/src/main/java/org/springframework/cloud/stream/schema/client/config/SchemaRegistryClientProperties.java

```diff
@@ -43,4 +45,6 @@ public boolean isEnabled() {
 	public void setEnabled(boolean enabled) {
```

Comment:
We can remove this property as it is not used in code. The usage within `@ConditionalOnProperties` depends on the presence (or absence) of the property in the environment.